.github/workflows/build-and-test.yml

```
@@ -20,63 +20,28 @@ jobs:
     name: Build and Test Swift Package iOS
     uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
     with:
-      scheme: TemplatePackage
-      artifactname: TemplatePackage.xcresult
-  packagewatchos:
-    name: Build and Test Swift Package watchOS
-    uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
-    with:
-      scheme: TemplatePackage
-      resultBundle: TemplatePackageWatchOS.xcresult
-      destination: 'platform=watchOS Simulator,name=Apple Watch Series 8 (45mm)'
-      artifactname: TemplatePackageWatchOS.xcresult
-  packagevisionos:
-    name: Build and Test Swift Package visionOS
-    uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
-    with:
-      xcodeversion: latest
-      scheme: TemplatePackage
-      resultBundle: TemplatePackageVisionOS.xcresult
-      destination: 'platform=visionOS Simulator,name=Apple Vision Pro'
-      artifactname: TemplatePackageVisionOS.xcresult
+      scheme: SpeziLicense
+      artifactname: SpeziLicense.xcresult
```

As we have multiple targets in the Swift Package you will need to use:
```suggestion
      scheme: SpeziLicense-Package
      artifactname: SpeziLicense-Package.xcresult
```

---

So this is also if we just have one target and one test-target? I just checked other repos like SpeziOnboarding, where it's the same case. There, the "-Package" suffix is not used, that's why I did it the same way.
But can change it of course.

---

You are completely right; I missed why this error is appearing here. I thought it was due to having multiple targets in the Swift Package (which is when the `-Package`) postfix needs to be added as Xcode generates one package scheme for all targets.

This error seems to stem from the fact that you have not renamed the package target but expected it to be called `SpeziLicense` while it is still called `TemplatePackage ` in the Package.swift file.

As we will close this PR, you can take these GitHub Actions and put them in the other branch and everything should work as expected as you have renamed the target there 👍